build.sbt

```diff
@@ -174,7 +160,6 @@ lazy val ratatoolCli = project
     name := "ratatool-cli",
     libraryDependencies ++= Seq(
       "com.github.scopt" %% "scopt" % scoptVersion,
-      "org.apache.parquet" % "parquet-avro" % parquetVersion,
```

Comment:
did we mean to remove this

Reply:
Yea I think it ended up there as an artifact during the project restructure but seems to work fine without